SPECS/ocaml-sexplib/ocaml-sexplib.spec

```diff
@@ -0,0 +1,337 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
+%endif
+
+%bcond_with vim
+
+%global srcname sexplib
+
+Name:           ocaml-%{srcname}
```

Comment:
I think you forgot to remove `SPECS-EXTENDED` version of this spec.

Reply:
removed